internal/config/config.go

```diff
@@ -99,9 +125,10 @@ func LoadConfig(filePath string, fileName string) (*Config, error) {
 
 	// Config File
 	v.AddConfigPath(filePath)
+	v.AddConfigPath(".")
```

Comment:
This is useful when running it locally, we were missing it 👍

Reply:
Now that I think about it.. should we create a const for it?

Reply:
I thought about it but then it would be an array/slice and slices are not constants in Go.

I accept suggestions.

Reply:
Done.

Reply:
I mean simply to add a separate constant like ` const localPath = "."` instead of an inline variable.

The other approach would be to use a func just for that:
```golang
func getSlice() []string {
    return []string{"hello", "world"}
}
```
but maybe it is too much